docs/sdk-and-tools/sdk-py/mxpy-cli.md

```diff
@@ -166,46 +315,17 @@ The _packaged source_ passed as `--packaged-src` can be obtained either from [th
+## Creating and sending transactions
 
-To convert a wallet from a type to another you can use:
+To create a new transaction we use the `mxpy tx new` command. Let's see how that works:
 ```sh
-mxpy wallet convert
-```
-
-The available formats are:
- - `raw-mnemonic` - secret phrase in plain text
- - `keystore-mnemonic` - secret phrase, as a password-encrypted JSON keystore file
- - `keystore-secret-key` - secret key (irreversibly derived from the secret phrase), as a password-encrypted JSON keystore file
- - `pem` - secret key (irreversibly derived from the secret phrase), as a PEM file
-
-:::info
-Keep in mind that the conversion isn't always possible (due to irreversible derivations of the secret phrase):
- - `raw-mnemonic` can be converted to any other format
- - `keystore-mnemonic` can be converted to any other format
- - `keystore-secret-key` can only be converted to `pem`
- - `pem` can only be converted to `keystore-secret-key`
-
-It's mandatory that you keep a backup of your secret phrase somewhere safe.
-:::
-
-For example, let's say you want to convert from _mnemonic_ to a _PEM_ file. We can do that by typing the following:
-```sh
-mxpy wallet convert --in-format raw-mnemonic --out-format pem --outfile key.pem
-```
-
-You will then be asked to enter the mnemonic. After typing the mnemonic press `Ctrl + D` for Linux/MacOS or `Ctrl + Z` for Windows.
-
-We can also convert your mnemonic to a keystore file. That can be done by typing:
-```sh
-mxpy wallet convert --in-format raw-mnemonic --out-format keystore-mnemonic --outfile keystore.json
+mxpy tx new --pem ~/multiversx-sdk/testwallets/latest/users/alice.pem --recall-nonce \
+    --receiver erd1spyavw0956vq68xj8y4tenjpq2wd5a9p2c6j8gsz7ztyrnpxrruqzu66jx \
+    --gas-limit 50000 --value 1000000000000000000 \
+    --proxy https://devnet-gateway.multiversx.com --chain D \
+    --send
 ```
 
-After inserting the mnemonic you will be asked to provide a password for the file. Insert the password then press enter. A file named `keystore.json` has been created.
+That's it! As easy as that. We sent a transaction from Alice to Bob. We choose the receiver of our transaction using the `--receiver` argument and setted the gas limit to `50000` because that is the gas cost of a simple move balance transaction. Notice we used the `--value` argument to pass the value that we want to transfer but we passed in the denomintated value. We transferred 1 eGLD (1 * 10^18). We then specify the proxy and the chain ID for the network we want to send our transaction to and use the `--send` argument to broadcast it.
 
-The command arguments can be found [here](https://github.com/multiversx/mx-sdk-py-cli/blob/main/CLI.md#walletconvert) or by typing:
-```sh
-mxpy wallet convert --help
-```
+In case you want to save the transaction you can also provide the `--outfile` argument and a `json` file containing the transaction will be saved at the specified location.
```

Comment:
maybe specify that the user should remove the `--send` argument?

Reply:
Added a new sentence to clarify.